magicctapipe/scripts/lst1_magic/lst_m1_m2_cloud_correction.py

```diff
@@ -107,46 +119,266 @@ def trans_height(x, Hc, dHc, trans):
     return t
 
 
-def process_telescope_data(input_file, config, tel_id, camgeom, focal_eff):
+def lidar_cloud_interpolation(
+    mean_subrun_timestamp, max_gap_lidar_shots, lidar_report_file
+):
+    """
+    Retrieves or interpolates LIDAR cloud parameters based on the closest timestamps to an input mean timestamp of the processed subrun.
```

Comment:
I think you should add in this docstring that for the moment in the case of multiple clouds only the one with the lowest transmission is taken into account